rosdoc2/verbs/build/builders/index.rst.jinja

```diff
@@ -1,7 +1,7 @@
 .. GENERATED_CONTENT by rosdoc2.verbs.build.builders.sphinx_builder.
 
-Welcome to the documentation for {{ package.name }}
-================================={% for _ in package.name %}={% endfor %}
+ROS2 Package <{{ package.name }}>
+=============={% for _ in package.name %}={% endfor %}=
```

Comment:
:grin: 

```suggestion
ROS 2 Package <{{ package.name }}>
==============={% for _ in package.name %}={% endfor %}=
```

Reply:
Yes, definitely.

But I might argue that we don't want to call these "ROS 2 packages" at all.  For instance, Gazebo and Fast-DDS both have package.xml files, but I wouldn't necessarily consider either of them to be a "ROS 2 package".

How about "Package documentation for {{package.name}}", or something like that?

Reply:
When rosdoc2 is used, aren't we presenting the documentation AS a ROS 2 package? Isn't gazebo_ros, when documented by rosdoc2, a representation of gazebo as a ROS 2 package?

I'm speaking as an amateur to experts here, so I really don't know the right answer, but I will argue for being as specific as reasonably possible. 'package' is such a generic term that "Package documentation for {{package.name}}" isn't saying much.

Reply:
For simplifying this I would actually suggest that we drop this down to just the package name. It ends up in several places you can see it here in the headline and the top navigation and the tab heading. 

![image](https://github.com/ros-infrastructure/rosdoc2/assets/447804/4062387b-61e3-4bdf-9868-51209f39303e)

If we don't put the package name at the front the tab becomes almost unusable. 

And re calling it a ROS package or not. That is moderately implicit based on where it's being documented or used. Like we don't put ROS in the package name. The fact that it's on a ROS website or from a ROS workspace that's enough for people to know that it's a ROS package. And we have written this documentation engine to be actually generic so it can be used for gazebo packages. Or potentially any package outside of the ecosystem if they want to use our standards.

Reply:
```suggestion
<{{ package.name }}>
={% for _ in package.name %}={% endfor %}=
```

Reply:
![image](https://github.com/ros-infrastructure/rosdoc2/assets/447804/65bc7b81-66de-4717-9b1c-1a25421d2ca8)
 simplified view with tab title, header all consistent